docs/references/files/fossa-yml.md

```diff
@@ -114,7 +114,10 @@ The name field sets the projects visible name in the FOSSA dashboard. By default
 The name of the team in your FOSSA organization to associate this project with.
 
 #### `project.policy:`
-The name of the policy in your FOSSA organization to associate this project with.
+The name of the policy in your FOSSA organization to associate this project with. Mutually excludes `project.policyId`.
+
+#### `project.policyId:`
+The id of the policy in your FOSSA organization to associate this project with. Mutually excludes `project.policy`.
```

Comment:
Please document the exact semantics, since these are both marked "mutually excludes".

Reading the code, I believe that if both are provided, `policy` takes precedence.
If this is correct, please clearly document everywhere we document these flags/settings.

I recommend using wording similar to:
```
#### `project.policy:`
The name of the policy in your FOSSA organization with which to associate this project. 

If this option and `project.policyId` are both provided, 
this option takes precedence and causes `project.policyId` to be ignored.

#### `project.policyId:`
The id of the policy in your FOSSA organization with which to associate this project. 

If this option and `project.policy` are both provided, 
`project.policy` takes precedence and causes this option to be ignored.
```

Reply:
I think it'd also be helpful to tell users how to get the ID of a given policy.

If we don't have existing Core-side docs to point them to for this, please consider writing a quick doc in `docs.fossa.com` to link to.

Reply:
I've added a note here about how the CLI variants of these values take precedence over config file ones. From our other discussion here I think that `mutually excludes` is the right language otherwise. 

I will look into making a doc tomorrow on where to get a policy id, right now the main way I know of to get them is from the URL of the policy page.